### DISCUSSION TITLE
2006/02/25/%E6%9E%84%E5%BB%BA%E5%8C%BB%E9%99%A2%E4%BF%A1%E6%81%AF%E7%B3%BB%E7%BB%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#75] [Announcements]
# 2006/02/25/%E6%9E%84%E5%BB%BA%E5%8C%BB%E9%99%A2%E4%BF%A1%E6%81%AF%E7%B3%BB%E7%BB%9F

自从我发现了STL，再编写C++程序，就离不开它了。STL提供的容器和算法极大的方便了C++编程。

https://blog.qizhen.xyz/2006/02/25/%E6%9E%84%E5%BB%BA%E5%8C%BB%E9%99%A2%E4%BF%A1%E6%81%AF%E7%B3%BB%E7%BB%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Jenny
2010年04月7日 21:39 编辑
软件开发：10人，六个月，￥1,000,000 这个估计是做梦把

回复
阿七
2009年11月20日 19:04 编辑
呵呵

回复
Qizhen
2006年03月31日 00:06 编辑
600w对于医院来说，根本就是九牛一毛马，有什么好大惊小怪的 :D

回复
palm
2006年03月30日 21:20 编辑
花600w搭建一个中小型的医院信息系统？作者是疯了还是太高高在上了啊。

回复
